addon/popup.js

```diff
@@ -185,11 +188,67 @@ class App extends React.PureComponent {
         isInSetup: locationHref.includes("/lightning/setup/"),
         contextUrl: locationHref,
       });
+      this.getListViewQuery(locationHref);
     }
     this.setState({
       isFieldsPresent: e.data.isFieldsPresent,
     });
   }
+  async getListViewQuery(url) {
+    if (localStorage.getItem("enableListViewExport") === "false") {
```

Comment:
Since we changed the default option, the test also needs to be updated
```suggestion
    if (localStorage.getItem("enableListViewExport") !== "true") {
```